packages/react-storage/src/components/StorageBrowser/controls/ActionCancelControl.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import { ControlProps } from './types';
+import { useResolvedComposable } from './hooks/useResolvedComposable';
+import { ActionCancel } from '../composables/ActionCancel';
+import { useActionCancel } from './hooks/useActionCancel';
+import { ViewElement } from '../context/elements';
+
+export const ActionCancelControl = ({
+  className,
+}: ControlProps): React.JSX.Element | null => {
```

Comment:
I don't think it would even be null?

Reply:
True, previously it would return null if there are no props but now it will always be there so I can remove this.